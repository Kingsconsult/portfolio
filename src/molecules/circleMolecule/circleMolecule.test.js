import React from 'react';
import renderer from "react-test-renderer";
import CircleDisplay  from "./index";
 
import { ThemeProvider } from "styled-components";
const theme = {
  mode: "light"
};


it("circles displays  correctly when called ", () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={theme}>
            <CircleDisplay />
      </ThemeProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});