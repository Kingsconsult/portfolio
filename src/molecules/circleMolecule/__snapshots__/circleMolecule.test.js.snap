// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cards renders correctly when called  1`] = `
<div
  className="sc-AxirZ gQzsnb"
  color="black"
  size="small"
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <h1
      className="sc-AxiKw jCCQlQ"
      size="big"
      style={
        Object {
          "marginTop": "5px",
        }
      }
    >
      1285
    </h1>
    <h1
      className="sc-AxiKw bDqPLV"
      size="small"
      style={
        Object {
          "marginTop": "-30px",
        }
      }
    >
      PROJECTS
    </h1>
  </div>
</div>
`;

exports[`circles displays  correctly when called  1`] = `
<div
  className="sc-AxirZ gQzsnb"
  color="black"
  size="small"
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <h1
      className="sc-AxiKw jCCQlQ"
      size="big"
      style={
        Object {
          "marginTop": "5px",
        }
      }
    >
      1285
    </h1>
    <h1
      className="sc-AxiKw bDqPLV"
      size="small"
      style={
        Object {
          "marginTop": "-30px",
        }
      }
    >
      PROJECTS
    </h1>
  </div>
</div>
`;

exports[`cirl renders correctly when called  1`] = `
<div
  className="sc-AxirZ gQzsnb"
  color="black"
  size="small"
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <h1
      className="sc-AxiKw jCCQlQ"
      size="big"
      style={
        Object {
          "marginTop": "5px",
        }
      }
    >
      1285
    </h1>
    <h1
      className="sc-AxiKw bDqPLV"
      size="small"
      style={
        Object {
          "marginTop": "-30px",
        }
      }
    >
      PROJECTS
    </h1>
  </div>
</div>
`;
